frontend/packages/integration-tests-cypress/tests/monitoring/monitoring.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { checkErrors, testName } from '../../support';
+import { submitButton, errorMessage } from '../../views/form';
+import { listPage } from '../../views/list-page';
+import { detailsPage } from '../../views/details-page';
+import { modal } from '../../views/modal';
+
+const shouldBeWatchdogAlertDetailsPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alert Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+const shouldBeWatchdogAlertRulesPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alerting Rule Details');
+  detailsPage.sectionHeaderShouldExist('Active Alerts');
+};
+
+const shouldBeWatchdogSilencePage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Silence Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+describe('Monitoring: Alerts', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  // each it() should be its own atomic test (run independently of other tests), per best practices.
+  it('displays and filters the Alerts list page, links to detail pages', () => {
+    cy.log('use vert. nav. menu to goto Monitoring -> Alerting');
+    cy.clickNavLink(['Monitoring', 'Alerting']);
+    // TODO, switch to 'listPage.titleShouldBe('Alerting');', when we switch to new test id
+    cy.byLegacyTestID('resource-title').should('have.text', 'Alerting');
+    listPage.projectDropdownShouldNotExist();
+
+    cy.log('filter Alerts');
+    listPage.filter.byName('Watchdog');
+    listPage.rows.countShouldBe(1);
+
+    cy.log('drills down to Alert details page');
+    listPage.rows.shouldExist('Watchdog').click();
+    shouldBeWatchdogAlertDetailsPage();
+
+    cy.log('drill down to the Alerting Rule details page');
+    cy.byTestID('alert-rules-detail-resource-link')
+      .contains('Watchdog')
+      .click();
+    shouldBeWatchdogAlertRulesPage();
+
+    cy.log('drill back up to the Alert details page');
+    // Active Alerts list should contain a link back to the Alert details page
+    cy.byTestID('active-alerts')
+      .first()
+      .click();
+    shouldBeWatchdogAlertDetailsPage();
+  });
+
+  it('creates and expires a Silence', () => {
+    // each it() should be its own atomic test, likely start with cy.visit(the-page) or nav to page
```

Comment:
Again, the README feels like the right spot for this comment, not necessarily here.

Reply:
fixed, updated README